monix/src/main/scala/com/avast/fairytale/functionk/monix/package.scala

```diff
@@ -1,11 +1,10 @@
-package com.avast.fairytale
+package com.avast.fairytale.functionk
 
 import cats.Eval
 import cats.arrow.FunctionK
-import monix.eval.Task
-
-package object functionk {
+import _root_.monix.eval.Task
```

Comment:
Can you remove `_root_`?

Reply:
It's easy. You can't have multiple package objects definitions.